lib/flatfile.ts

```diff
@@ -409,7 +373,7 @@ export const addDocumentToSpace = async (
   return addDocumentResult.data;
 };
 
-export interface BlueprintWithId extends Blueprint {
+export interface BlueprintWithId {
   id: string;
 }
```

Comment:
If we're taking away blueprint, this is just `{ id: string}` and doesn't need a custom type

Reply:
removed